src/apollo/client.ts

```diff
@@ -18,6 +18,20 @@ const authLink = setContext((_, { headers }) => {
   };
 });
 
+const cache = new InMemoryCache({
```

Comment:
We can have typed cache policy:
https://www.graphql-code-generator.com/docs/plugins/typescript-apollo-client-helpers

Reply:
Added but in this case with local schema field it won't help much.

Reply:
Thought its smart enough to handle local schema too :(